src/validation.h

```diff
@@ -127,7 +127,7 @@ static const unsigned int MAX_BLOCKS_TO_ANNOUNCE = 8;
 /** Maximum number of unconnecting headers announcements before DoS score */
 static const int MAX_UNCONNECTING_HEADERS = 10;
 
-static const bool DEFAULT_PEERBLOOMFILTERS = true;
+static const bool DEFAULT_PEERBLOOMFILTERS = false;
```

Comment:
This has nothing to do with validation, please move it to net or something.

Reply:
agree on that probably being better in different pr

Reply:
No, we already have more than enough volume of useless "move code around" PRs. This is precisely the kind of thing that should be in the same PR in a separate commit.

Reply:
let's not shedpaint this but imo it's fine to ombine this small code structure improvement with the change